### ISSUE TITLE
Plugin List not populating in Mura 7.0.7077

### ISSUE THREAD

**Issue by JasPanesar** [nolanerck/Mura-IFrame-Plugin#2]
When I go to access the Content > Layout > Content Objects > Plugins , the Plugin dropdown does not populate with the iframe site I have successfully added.

Open to trying out any ideas.  Video available if needed.

Plugin was installed Mura 7.0.7077 using current Chrome and Firefox.

**Comment by nolanerck** [comment]
@JasPanesar yes please send the video. It's been quite a while since I've looked at this plugin, seeing what you're dealing with might help refresh my memory. :)
